portfolio/src/main/java/com/google/sps/servlets/Country.java

```diff
@@ -0,0 +1,11 @@
+package com.google.sps.servlets;
+
+class Country {
+    private String country;
```

Comment:
You can declare these fields `final` since they're only set once.

You probably also want to add `get` methods at some point.